project_task_note/views/project_task_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--
+        Copyright 2024 Tecnativa Carolina Fernandez
+        License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+    -->
+    <!-- Project Task form -->
+    <record id="view_task_form2" model="ir.ui.view">
+        <field name="name">project_task.form.note</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <page name="description_page" position="after">
```

Comment:
I'm thinking that is better to put the notes in the same tab of the description, as being in read mode, you don't have to switch between pages to see both texts, and they are shortened to the minimum occupied height. Anyway, check how this will see in v16 (always in edit mode).